packages/video_player/video_player/lib/video_player.dart

```diff
@@ -290,6 +297,7 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
           initializingCompleter.complete(null);
           _applyLooping();
           _applyVolume();
+          _applyPlaybackSpeed();
```

Comment:
Since PlaybackSpeed only gets applied if the video is playing, should this be called after `_applyPlayPause()`?

Reply:
Also: _applyPlayPause is already calling _applyPlaybackSpeed, do we need it here?

Reply:
You are totally right!